Source/JavaScriptCore/runtime/JSGenericTypedArrayViewInlines.h

```diff
@@ -1013,6 +1013,17 @@ template<typename PassedAdaptor> inline Structure* JSGenericResizableOrGrowableS
     return Structure::create(vm, globalObject, prototype, TypeInfo(typeForTypedArrayType(Base::Adaptor::typeValue), StructureFlags), info(), NonArray);
 }
 
+template<typename PassedAdaptor> inline bool JSGenericResizableOrGrowableSharedTypedArrayView<PassedAdaptor>::preventExtensions(JSObject* cell, JSGlobalObject* globalObject)
```

Comment:
Just a question; How are `ALWAYS_INLINE` and `inline` used differently?

Reply:
`inline` is just controlling visibility and linking. In fact, it is not guaranteed that it will be actually inlined by the compiler: compiler decides whether it should be inlined or not based on compiler's modeling.
But `ALWAYS_INLINE` enforces inlining.